modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/ipc.dm

```diff
@@ -9,6 +9,11 @@
 	generic = "Screen"
 	relevent_layers = list(BODY_ADJ_LAYER)
 
+/datum/sprite_accessory/screen/is_hidden(mob/living/carbon/human/H, obj/item/bodypart/HD)
+	if((H.wear_mask && (H.wear_mask.flags_inv & HIDEEYES)) || !HD)
```

Comment:
Don't use single-letter vars.

Reply:
i knooow, but this is how they are written on every sprite_accessories dm, and on the proc Azarak made 😔